pkg/network/usm/ebpf_ssl.go

```diff
@@ -697,7 +698,7 @@ func addHooks(m *manager.Manager, procRoot string, probes []manager.ProbesSelect
 						continue
 					}
 				}
-				manager.SanitizeUprobeAddresses(elfFile, []elf.Symbol{sym})
+				uprobes.SanitizeAddresses(elfFile, []safeelf.Symbol{sym})
```

Comment:
how is the change of `manager.SanitizeUprobeAddresses` to `uprobes.SanitizeAddresses` relevant to the PR?

Reply:
`manager.SanitizeUprobeAddresses` uses the `elf.*` types, whereas `uprobes.SanitizeAddresses` accepts the `safeelf.*` types.

Reply:
Where possible, I wanted to encourage the use of the `safeelf` package and types. Escaping to the underlying types at every chance detracts from that goal.

Reply:
I think my comment here suggests a simpler method https://github.com/DataDog/datadog-agent/pull/30725#discussion_r1828488018

Reply:
It is already a type alias. The goal was to encourage use of the `safeelf` package and types. If you see that package in the arguments, you are more likely to understand you should use that package rather than `debug/elf`.

Reply:
then let's find a cleaner solution like changing the manager SanitizeAddress to get `symbols ...elf.Symbol` instead of an array

There is no need to duplicate functions from the manager package and possibly create forks in implementation of the same function